pkgs/applications/editors/vscode/extensions/vscode-utils.nix

```
@@ -88,7 +88,7 @@ let
         "vsix"
       ])
       // {
-        name = "${mktplcRef.publisher}-${mktplcRef.name}-${mktplcRef.version}";
+        name = "${mktplcRef.publisher}-${mktplcRef.name}";
```

```suggestion
        pname = "${mktplcRef.publisher}-${mktplcRef.name}";
```

Shouldn't this be ^^ ?

---

Yes, but that would be a breaking change for the `buildVscodeExtension` function.

---

Fair enough.

---

We could move this to after branch-of, add a changelog entry.

---

Sure why not!